### PR TITLE
Week 12 email

### PR DESCRIPTION
**Pull request by jrtechs**
Week 12 email. Wow, this has a bunch of things in it.

### CODE REVIEW

**Comment by nlMeminger** [comment]
Should we mention more about Anthony? give a bit more info about him and such?

**Comment by Tjzabel** [comment]
@nlMeminger can you offer any more insight on what to add about Anthony? I don't know much about him.

**Comment by nlMeminger** [comment]
he works at Datto, used to work at RIT for the IST department.  Got his undergrad and grauate degrees from RIT

File: announcements/_posts/2019-11-15-w12-kernel-tracing.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Looking forward to seeing you all at this weeks meeting,
+
+- Jeffery Russell (@JrTechs)
+Fellow RITlugger, best friend, and your reindeer to guide the sleigh
+
+**Upcoming Events:**
+
+* [FOSS Family Dinner at Bar Louie](https://forms.gle/Tyrdu1U53Wyay1QeA): November 21st
+* [Rochester Maker Faire](https://rochester.makerfaire.com/): November 23rd
+* FOSS Hours: Tuesdays 5:00-7:00pm Magic Center 3190
```
Comment (Tjzabel) [inline announcements/_posts/2019-11-15-w12-kernel-tracing.md]: So close! *Thursdays*

**Comment by Tjzabel** [comment]
@nlMeminger can you come up with an official blurb to place there?

**Comment by nlMeminger** [comment]
I just reached out to him for help on that

**Comment by nlMeminger** [comment]
I posted in slack what was sent to me from Anthony.  just copy pasta his bio from the redhat site and all that

File: announcements/_posts/2019-11-15-w12-kernel-tracing.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+title: "Kernel Tracing"
+layout: post-event
+date-start: "2019-10-18 16:30"
+date-end: "2019-10-18 18:00"
+location: "GOL-2620 (Med. DB Lab)
+---
+
+Hey RITluggers,
+
+This week we have Anthony Critelli visiting our club to give a special presentation on kernel tracing.
```
Comment (nlMeminger) [inline announcements/_posts/2019-11-15-w12-kernel-tracing.md]: Please add bio about anthony from here https://www.redhat.com/sysadmin/users/acritelli

**Comment by Tjzabel** [comment]
@jrtechs @nlMeminger I added more to Anthony's bio, and fixed the FOSS Hours date.

**Comment by nlMeminger** [comment]
good for me

**Comment by nlMeminger** [comment]
happy now @Tjzabel ?

**Comment by Tjzabel** [comment]
@nlMeminger yep. Just wanted to revoke the stale review to make sure we're all on the same page.

**Comment by jrtechs** [comment]
Thanks @nlMeminger @Tjzabel

**Pull request merged**